salt/_modules/metalk8s_network.py

```diff
@@ -0,0 +1,45 @@
+'''Metalk8s network related utilities.'''
+
+import ipaddress
```

Comment:
`New in version 3.3.`

I thought we were stuck to Python 2 for Salt (see this [reply](https://github.com/scality/metalk8s/issues/780#issuecomment-528441446)) ?
Or are we somehow magically pulling the backport to 2.X from pip or something?

Reply:
Looks like the module is available by default in python2.

Here: 
```
[root@bootstrap vagrant]# python
Python 2.7.5 (default, Oct 30 2018, 23:45:53) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipaddress
>>> 
```

Reply:
No it's not _available by default_, but actually comes from a backport as @slaperche-scality suggested: https://pypi.org/project/ipaddress/

However, this module is `Required-by` the `docker` client, which you probably have installed on your machine. We could add the dependency to the project (as we do with `m2crypto` for instance), or move back to relying on a more manual approach (using `__utils__` for instance).

Reply:
Now, Salt uses this as well, relying on their `salt._compat` layer: [see this import in the `salt.utils.network` module](https://github.com/saltstack/salt/blob/develop/salt/utils/network.py#L30). Let's do the same, should be sufficient:

```suggestion
from salt._compat import ipaddress
```